pyEpiabm/pyEpiabm/utility/state_transition_matrix.py

```diff
@@ -0,0 +1,65 @@
+#
+# Generate state transmission matrix
+#
+
+import pandas as pd
+import numpy as np
+from pyEpiabm.property import InfectionStatus
+
+
+class StateTransitionMatrix:
+    """Class to generate the state transition matrix
+    """
+    def build_state_transition_matrix(self):
+        """Builds the structure of the state transition matrix that is used in
+        the host progression sweep. Labels the rows and the columns with the
+        infection status. Fill the rest of the dataframe with zero
+        probabilities of transition.
+
+        :returns: Symmetric matrix in the form of a dataframe
+        :rtype: Pandas dataframe
+        """
+        # Currently, method very rigid. Can add flexibility later.
+        nb_states = len(InfectionStatus)
+        zero_trans = np.zeros((nb_states, nb_states))
+        labels = ['Susceptible', 'Exposed', 'InfectASympt', 'InfectMild',
+                  'InfectGP', 'InfectHosp', 'InfectICU', 'InfectICURecov',
+                  'Recovered', 'Dead']
+        init_matrix = pd.DataFrame(zero_trans,
+                                   columns=labels,
+                                   index=labels)
+        return init_matrix
+
```

Comment:
Would we expect to be able to change these transition probabilites (ie new varient?). Maybe another quick function which can change a specific value